### DISCUSSION TITLE
Disable the /openapi.json text from docs page.

### DISCUSSION THREAD

**Discussion by sailikpandey22** [fastapi/fastapi#11322] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import FastAPI

app = FastAPI()

@app.get("/")
async def read_root():
   return {"message": "Hello User!"}
```

### Description

http://127.0.0.1:8000/docs
in this endpoint I want to hide the `/openapi.json` and `OAS 3.1` text
<img width="560" alt="Screenshot 2024-03-20 at 11 08 12 PM" src="https://github.com/tiangolo/fastapi/assets/102276321/03a5e4bc-fc66-496d-853c-9017c47db63a">

### Operating System

macOS

### FastAPI Version

0.109.2

### Pydantic Version

2.6.2

### Python Version

Python 3.11.0

**Comment by jgould22** [comment, marked as answer]
You will probably need to download the swagger doc and edit the page to remove those.

You can start here https://fastapi.tiangolo.com/how-to/custom-docs-ui-assets/#self-hosting-javascript-and-css-for-docs

Keep in mind if you serve it yourself you will have to update it yourself since it wont come from the CDN as it currently does

**Reply by sailikpandey22** [reply]
Can you please help me out about what to edit. I read that documentation and now i am serving static file from the static folder. but i want to remove that `/openapi.json` . I am not that pro in web dev actually..

**Reply by JavierSanchezCastro** [reply]
I don't know why you want to remove that text from the Swagger, in any case, just add 
``visibility: hidden``
in
``.swagger-ui .info .title small.version-stamp``

Regarding deleting the openapi.json, if you add ``include_in_schema=False``in each endpoint, you will not see anything there. Again, I don't know if that has any utility.

**Reply by sailikpandey22** [reply]
Thanks though I solved it myself :) 
Its looking great. a small project made with fastapi https://api.thefutureproject.tech/docs

**Reply by SahaanaVenkat** [reply]
@sailikpandey22  May I know how did you resolved even I am also facing same requirement
